tools/proofers/for_mentors.php

```diff
@@ -243,14 +244,14 @@ function page_summary_sql($mentored_round, $projectid)
             ELSE CONCAT('<a href=\""
                 .$code_url . "/stats/members/mdetail.php?&id=',u.u_id,
                 '\">',u.username,'</a>')
-            END AS '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Proofreader")) . "',
-            COUNT(1) AS '" . mysqli_real_escape_string(DPDatabase::get_connection(), _("Pages this project")) . "',
+            END AS '" . DPDatabase::escape(_("Proofreader")) . "',
+            COUNT(1) AS '" . DPDatabase::escape(_("Pages this project")) . "',
```

Comment:
sprintf perhaps?

Reply:
I think in this case it would end up being confusing. There's a lot of interleaving of field names (using variable interpolation) which we can't directly `sprintf` because the strings might contain `%`s.

Ironically, I think the more readable solution in this case is to introduce some variables like the comment below so we don't need to have all the escaping and concatenation inline. Let me know if you dislike it.